ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -54,19 +54,29 @@ tests:
     make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
   container:
     from: test-bin
-- as: e2e-aws
+- as: e2e
   steps:
-    cluster_profile: aws
-    workflow: openshift-e2e-aws-loki
-- as: e2e-gcp-upgrade
+    cluster_profile: gcp
+    workflow: openshift-e2e-gcp-loki
```

Comment:
I picked GCP to back the new `e2e`.  Let me know if you prefer AWS.

Reply:
no strong feelings